backend/sim/wallet/address.go

```diff
@@ -113,11 +102,11 @@ func (a *Address) Cmp(addr wallet.Address) int {
 	return yCmp
 }
 
-// Encode encodes this address into an io.Writer. Part of the
-// go-perun/pkg/io.Serializer interface.
-func (a *Address) Encode(w io.Writer) error {
+// MarshalBinary marhals the address into a binary form.
```

Comment:
marshals

Reply:
Updated in ec78a5b3c78f2347f4270b35231a1b7060af3467.